src/lib/logger/impl.mli

```diff
@@ -11,7 +11,16 @@ end
 type t = Stable.V1.t
 
 module Level : sig
-  type t = Spam | Trace | Debug | Info | Warn | Error | Faulty_peer | Fatal
+  type t =
+    | Spam
+    | Trace
+    | Debug
+    | Info
+    | Warn
+    | Error
+    | Faulty_peer
+    | Fatal
+    | Internal
```

Comment:
Nit: it might be cleaner to call this log level `Checkpoint` or similar, `Internal` feels somewhat non-specific.

Reply:
@psteckler what's your preference between `Checkpoint` and `Internal`?